test/contracts/solidity/L1L2.sol

```diff
@@ -2,14 +2,14 @@
 pragma solidity ^0.6.12;
 
 /**
-  Demo contract for L1 <-> L2 interaction between an L2 StarkNet contract and this L1 solidity
+  Demo contract for L1 <-> L2 interaction between an L2 Starknet contract and this L1 solidity
```

Comment:
Changing the content of smart contract files might change some expected addresses and hashes, so I'd leave them intact.